backend/src/main/java/gov/cdc/usds/simplereport/service/TestOrderService.java

```diff
@@ -598,9 +598,13 @@ public OrganizationLevelDashboardMetrics getOrganizationLevelDashboardMetrics(
 
     List<AggregateFacilityMetrics> facilityMetrics = new ArrayList<AggregateFacilityMetrics>();
 
+    // we don't seem to be using the "organizationLevelDashboardMetrics" query anywhere so we can
+    // probably clean up this method
+    String diseaseLoinc = "96741-4";
```

Comment:
we don't seem to be using the "organizationLevelDashboardMetrics" query anywhere so we can probably clean up this method